### PR TITLE
ssl: Add missing specs for API functions

### PR DESCRIPTION
**Pull request by IngelaAndin**
Update so that all API functions uses specs

### CODE REVIEW

File: lib/ssl/src/ssl_crl_cache.erl [hunk 1]
```diff
@@ -72,9 +72,20 @@ fresh_crl(#'DistributionPoint'{distributionPoint = {fullName, Names}}, CRL) ->
 %% API 
 %%====================================================================
 
+%%--------------------------------------------------------------------
+-spec insert(CRLSrc) -> ok | {error, Reason} when
+      CRLSrc :: crl_src(),
+      Reason :: term().
+%%--------------------------------------------------------------------
 insert(CRLs) ->
     insert(?NO_DIST_POINT, CRLs).
```
Comment (u3s) [inline lib/ssl/src/ssl_crl_cache.erl:81]: If not a big deal, maybe align naming have either CRLSrc or CRLs depending on which is better.

File: lib/ssl/doc/src/ssl_crl_cache.xml [hunk 2]
```diff
@@ -41,36 +41,23 @@
...
-	  <v> CRLSrc = <seetype marker="#crl_src">crl_src()</seetype>]}</v>
-	  <v> URI = <seetype marker="#uri">uri()</seetype> </v>
-	  <v> Reason = term()</v>
-	</type>
+	<name since="OTP 18.0" name="insert" arity="1"></name>
+	<name since="OTP 18.0" name="insert" arity="2"></name>
+	<fsummary> Insert CRLs into the ssl applications local cache.</fsummary>
 	<desc>
-	 <p>Insert CRLs, available to fetch on DER format from <c>URI</c>, into the ssl applications local cache. </p>
+	  <p>Insert CRLs into the ssl applications local cache, with or without a distribution point
```
Comment (u3s) [inline lib/ssl/doc/src/ssl_crl_cache.xml:59]: Description does not refer to CRLSrc argument. Consider improving a little (it is about CRLs taken from CRLSrc I guess)

**Review by u3s — APPROVED** [review]
I'm fine with this change in general. Minor improvements can be considered if you agree and have time.

File: lib/ssl/src/ssl_crl_cache.erl (hunk 1 above)
Comment (IngelaAndin) [reply lib/ssl/src/ssl_crl_cache.erl]: Made a change

File: lib/ssl/doc/src/ssl_crl_cache.xml (hunk 2 above)
Comment (IngelaAndin) [reply lib/ssl/doc/src/ssl_crl_cache.xml:59]: See new names

**Pull request merged**